docs/project/contribute.md

```diff
@@ -26,11 +26,26 @@ Once on a file, click the 'pencil' icon to [easily edit the file inline](https:/
 ## Translations
 [![Crowdin](https://d322cqt584bo4o.cloudfront.net/all-contributors/localized.svg)](https://crowdin.com/project/all-contributors)
 
-Our translations are managed through crowdin. You can help contribute by heading to [the crowdin project](https://crowdin.com/project/all-contributors) and suggesting translations.
+Our translations are managed through crowdin. You can help contribute by heading to [the crowdin project] and suggesting translations.
```

Comment:
[minor] I'm not sure this works on the website

Reply:
It should as in Markdown, `[abc]` will be the same as any `[abc](efg)` where `[abc]` was defined as a URL pointing to `efg` with `[abc]: efg`.

Not sure if that was clear but I hope you got it and that crowdin's markdown renderer works like others in that sense.